src/components/accountsBottomSheet/container/accountsBottomSheetContainer.tsx

```
@@ -126,11 +128,11 @@ const AccountsBottomSheetContainer = () => {
       let _currentAccount = await switchAccount(accountData.username);
       let realmData = await getUserDataWithUsername(accountData.username);
 
-      _currentAccount.username = _currentAccount.name;
+      _currentAccount.name = _currentAccount.name;
```

Biome (2.1.2)

[error] name is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)